packages/jest-worker/src/Farm.ts

```diff
@@ -19,24 +20,25 @@ export default class Farm {
   private _computeWorkerKey: FarmOptions['computeWorkerKey'];
   private _cacheKeys: {[key: string]: WorkerInterface};
   private _callback: Function;
-  private _last: Array<QueueChildMessage>;
+  private _last: Array<QueueItem>;
   private _locks: Array<boolean>;
   private _numOfWorkers: number;
   private _offset: number;
-  private _queue: Array<QueueChildMessage | null>;
+  private _queue: Array<QueueItem | null>;
```

Comment:
```suggestion
  private _queue: Array<QueueItem>;
```

right? there can never be `null`s in the queue now?

Reply:
The queue can be empty, and that's signaled by a `null` item.

Reply:
so `Array<QueueItem> | [null]`? I guess it doesn't matter in practice

Reply:
The array index refers to each of the workers (I.e. `this._queue.length` is `numOfWorkers`, not the list of jobs); so each item in the array is either the first element of the linked list, or null.